WheatonApp/src/main/java/com/wheaton/app/HomeFragment.java

```diff
@@ -1,40 +1,45 @@
 package com.wheaton.app;
 
-import java.io.StringReader;
-import java.text.DateFormatSymbols;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Calendar;
-import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 
 import org.json.JSONArray;
 import org.json.JSONException;
-import org.xmlpull.v1.XmlPullParser;
-import org.xmlpull.v1.XmlPullParserException;
-import org.xmlpull.v1.XmlPullParserFactory;
 
+import android.content.Context;
 import android.graphics.Typeface;
 import android.os.Bundle;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.util.DisplayMetrics;
 import android.util.Log;
+import android.view.Display;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.ImageView;
 import android.widget.ListView;
 import android.widget.TextView;
 
-import com.wheaton.app.List.Header;
 import com.wheaton.app.List.Item;
 import com.wheaton.app.List.ListItem;
 import com.wheaton.app.List.TwoTextArrayAdapter;
 import com.wheaton.utility.LoadURLTask;
+import com.wheaton.utility.Utils;
+
 
 public class HomeFragment extends TrackedFragment {
-	
-	public HomeFragment() {
-        // Empty constructor required for fragment subclasses
+	private Context myContext;
+
+	public HomeFragment(Context aContext) {
```

Comment:
This will cause the app to crash, you can't pass parameters to the constructor of a Fragment.